### ISSUE TITLE
WARNING! You must upgrade to the latest version of google-spreadsheet!

### ISSUE THREAD

**Issue by yoyo930021** [revolunet/spreadsheet-to-json#14]
```
Google's deprecation date for the v3 sheets API is March 3rd 2020
Bad news - this version of this module will stop working on that date :(
Good news - the new version of the module uses the newer v4 api :)
However, there are breaking changes, so please see the docs site
https://theoephraim.github.io/node-google-spreadsheet
```
when using on now.

**Comment by poteat** [comment]
Yeah, ran into this issue myself.  Unfortunately I suspect this project is abandoned...

**Comment by revolunet** [comment]
nope, i just missed the PR #15 . checkin it

**Comment by revolunet** [comment]
i created a new PR which should fix it : https://github.com/revolunet/spreadsheet-to-json/pull/18

**Comment by revolunet** [comment]
released as 2.0.0
